docs/configure_open_edx.md

```diff
@@ -79,9 +79,9 @@ There are two options for this:
 - In devstack, run `make lms-shell` and within that shell `pip install mitxpro-openedx-extensions-$VERSION.tar.gz`
   - To update to a new development version without having to actually bump the package version, simply `pip uninstall -y mitxpro-openedx-extensions`, then install again
 
-### Configure xPro as a OAuth provider for Open edX
+### Configure mitX Online as a OAuth provider for Open edX
 
-In xPro:
+In mitX Online:
 
 - go to `/admin/oauth2_provider/application/` and create a new application with these settings selected:
   - `Redirect uris`: `http://<EDX_HOSTNAME>:18000/auth/complete/mitxpro-oauth2/`
```

Comment:
should this redirect uri be changed to `http://<EDX_HOSTNAME>:18000/auth/complete/mitxonline-oauth2/` or something similar?

Reply:
oh nvm it looks like these are set by the `social-auth-mitxpro`? is that right?